### DISCUSSION TITLE
popupinfo and consult-completion-in-region?

### DISCUSSION THREAD

**Discussion by OlMon** [minad/corfu#429] [Q&A] [answered]
Greetings,

Thanks for all the fantastic packages.

Not sure in which repo this question should go, so please forgive me if this is not the right place.

I often switch completion to minibuffer (maybe I switch completely to consult-completion-in-region) one thing that I would probably miss is the corfu-popupinfo functionality. Is there a way to have this functionality while using the minibuffer?
Maybe through Embark somehow?

Thanks a lot for your time.

Cheers,
OlMon

**Comment by minad** [comment, marked as answer]
Right now we don't have equivalent functionality to `corfu-popupinfo` for `consult-completion-in-region`.
